lib/checklists/question.rb

```diff
@@ -34,10 +34,16 @@ def possible_values
     (options + sub_options).map(&:value).compact
   end
 
+  def self.load(params)
+    parsed_params = params.dup
+    parsed_params['text'] = params['question']
+    parsed_params['options'] = Option.load_all(params['options'])
+    parsed_params['type'] = params['question_type']
```

Comment:
Same here in the future we could rename `question -> text` and `question_type -> type`

Reply:
Yup :-).